Proxy.env.template

```diff
@@ -0,0 +1,4 @@
+# Name of the instance, e.g. "myinstance" for http://myinstance.matrixreq.com
+INSTANCE=myinstance
+# The location of the plugin js file
+SCRIPT_PATH=dist/Main.js
```

Comment:
dist should be there as it's end up to be ../dist/dist/Main.js because of this line : 
```
thenFromFile(200, `../dist/${scriptFile}`)

```